ghost/core/test/e2e-api/admin/__snapshots__/settings.test.js.snap

```diff
@@ -694,6 +706,18 @@ Object {
       "key": "pintura_css_url",
       "value": null,
     },
+    Object {
+      "key": "donations_enabled",
+      "value": false,
+    },
+    Object {
+      "key": "donations_currency",
+      "value": "USD",
+    },
+    Object {
+      "key": "donations_suggested_amount",
+      "value": "0",
```

Comment:
Unsure why snapshots have `"0"` (string) instead of `0` (number) for `donation_suggested_amount`

Reply:
We don't cast values to the number type, not sure why https://github.com/TryGhost/Ghost/blob/81ef2ade39e57c8431585d971ba6c601278441a3/ghost/core/core/server/models/settings.js#L139-L160